setup.cfg

```diff
@@ -0,0 +1,28 @@
+[metadata]
+name = libindic-ngram
+summary = N-Gram generator for Indic Languages
+description-file =
+  README.md
+author = Santhosh Thottingal
+author-email = santhosh.thottingal@gmail.com
```

Comment:
@copyninja What about `Indic Project`? I think we can have something like `contact@indicproject.org` as the email ID also.

@anivar @stultus